commit 813e2197aa3ba43d5a0a959906882031834f1e76
Author: David Hummel <6109326+hummeltech@users.noreply.github.com>
Date:   Fri May 8 11:25:59 2020 -0700

    Fixed issue causing build to fail:
    
    ```
    In file included from src/json/mapnik_json_generator_grammar.cpp:25:0:
    include/mapnik/json/properties_generator_grammar_impl.hpp: In
    constructor ‘mapnik::json::properties_generator_grammar<OutputIterator,
    KeyValueStore>::properties_generator_grammar()’:
    include/mapnik/json/properties_generator_grammar_impl.hpp:71:27: error:
    ‘boost::phoenix::at_c’ has not been declared
         using boost::phoenix::at_c;
    ```

diff --git a/include/mapnik/json/properties_generator_grammar_impl.hpp b/include/mapnik/json/properties_generator_grammar_impl.hpp
index 8f694f6ca..1932c8d56 100644
--- a/include/mapnik/json/properties_generator_grammar_impl.hpp
+++ b/include/mapnik/json/properties_generator_grammar_impl.hpp
@@ -26,6 +26,7 @@
 #pragma GCC diagnostic push
 #include <mapnik/warning_ignore.hpp>
 #include <boost/spirit/include/phoenix.hpp>
+#include <boost/phoenix/fusion/at.hpp>
 #include <boost/fusion/adapted/std_tuple.hpp>
 #pragma GCC diagnostic pop
 
